libraries/testing/tester.cpp

```diff
@@ -199,9 +199,9 @@ namespace eosio::testing {
       open(snapshot);
    }
 
-   void base_tester::init(controller::config config, const genesis_state& genesis) {
```

Comment:
We may prefer an `enum class do_statrtup_t { no, yes }` instead of the bool here.

Reply:
I'll make the changes to use `enum class do_statrtup_t { no, yes }`.

Reply:
Done by https://github.com/AntelopeIO/spring/pull/133/commits/d08e3aec219a5b8f835b35ab3c911ab5803ef947